types/ityped/index.d.ts

```diff
@@ -1,15 +1,22 @@
-// Type definitions for ityped 0.0
+// Type definitions for ityped 1.0
 // Project: https://github.com/luisvinicius167/ityped
 // Definitions by: Daniel Rosenwasser <https://github.com/DanielRosenwasser>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 export interface Configuration {
     strings?: string[];
     typeSpeed?: number;
-    pause?: number;
+    backSpeed?: number;
+    backDelay?: number;
+    startDelay?: number;
+    cursorChar?: string;
+    placeholder?: boolean;
+    showCursor?: boolean;
+    disableBackTyping?: boolean;
+    onFinished?: () => unknown;
```

Comment:
```suggestion
    onFinished?: () => void;
```

Since the value is never observed, I think `() => void` might be slightly more appropriate here.